crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -1347,6 +1351,11 @@ pub(super) enum EventsPostProcessing {
     None,
 }
 
+pub(super) enum EventLocation {
+    InMemory,
+    InStore,
```

Comment:
Can these be called `Memory` and `Store`, only? Would be more concise, and the enum's name already indicates it's a location. (Otherwise it's temping to read it as "where is the event location itself?".)

Reply:
Sure. Done.